src/client/dom-processor/client-dom-adapter.ts

```diff
@@ -89,8 +89,8 @@ export default class ClientDomAdapter extends BaseDomAdapter {
         }
     }
 
-    getProxyUrl (): string {
-        return getProxyUrl.apply(null, arguments);
+    getProxyUrl (...args): string {
+        return getProxyUrl.apply(null, args as [string, any?]);
```

Comment:
Maybe we can just ensure type in the function arguments directly? There is also a TSLint rule in TestCafe that requires all function arguments to be typed.